docs/source/changelog/0.19.0-changelog.rst

```diff
@@ -0,0 +1,587 @@
+
+   - ``ManimColor.from_hex(hex=...)`` is now  ``ManimColor.from_hex(hex_str=...)``
+   - ``Scene.next_section(type=...)`` is now ``Scene.next_section(section_type=...)``
+
+
+* :pr:`3922`: Removed ``inner_radius`` and ``outer_radius`` from :class:`.Sector` constructor
+   To construct a :class:`.Sector`, you now need to specify a ``radius`` (and an ``angle``).
+   In particular, :class:`.AnnularSector` still accepts both ``inner_radius`` and ``outer_radius``
+   arguments.
+
+
+* :pr:`3964`: Allow :class:`.SurroundingRectangle` to accept multiple Mobjects
+   This changes the signature of :class:`.SurroundingRectangle` to accept
+   a sequence of Mobjects instead of a single Mobject. As a consequence, other
+   arguments that could be specified as positional ones before now need to be
+   specified as keyword arguments::
+
+      SurroundingRectangle(some_mobject, RED, 0.3)  # raises error now
+      SurroundingRectangle(some_mobject, color=RED, buff=0.3)  # correct usage
+
+
+* :pr:`4115`: Completely rewrite the implementation of the :class:`.Code` mobject
+   This includes several breaking changes to the interface of the class to make it
+   more consistent. See the documentation of :class:`.Code` for a detailed description
+   of the new interface, and the description of the pull request :pr:`4115` for
+   an overview of changes to the old keyword arguments.
+
+
+New features
+------------
+
+* :pr:`3148`: Added a ``colorscale`` argument to :meth:`.CoordinateSystem.plot`
+
+
+* :pr:`3612`: Add three animations that together simulate a typing animation
+
+
+* :pr:`3754`: Add ``@`` shorthand for :meth:`.Axes.coords_to_point` and :meth:`.Axes.point_to_coords`
+
+
+* :pr:`3876`: Add :meth:`.Animation.set_default` class method
+
+
+* :pr:`3903`: Preserve colors of LaTeX coloring commands
+
+
+* :pr:`3913`: Added :mod:`.DVIPSNAMES` and :mod:`.SVGNAMES` color palettes
+
+
+* :pr:`3933`: Added :class:`.ConvexHull`, :class:`.ConvexHull3D`, :class:`.Label` and :class:`.LabeledPolygram`
+
+
+* :pr:`3992`: Add darker, lighter and contrasting methods to :class:`.ManimColor`
+
+
+* :pr:`3997`: Add a time property to scene (:attr:`.Scene.time`)
+
+
+* :pr:`4039`: Added the ``delay`` parameter to :func:`.turn_animation_into_updater`
+
+
+Enhancements
+------------
+
+* :pr:`3829`: Rewrite :func:`~.bezier.get_quadratic_approximation_of_cubic` to produce smoother animated curves
+
+
+* :pr:`3855`: Log execution time of sample scene in the ``manim checkhealth`` command
+
+
+* :pr:`3888`: Significantly reduce rendering time with a separate thread for writing frames to stream
+
+
+* :pr:`3890`: Better error messages for :class:`.DrawBorderThenFill`
+
+
+* :pr:`3893`: Improve line rendering performance of :class:`.Cylinder`
+
+
+* :pr:`3901`: Changed :attr:`.Square.side_length` attribute to a property
+
+
+* :pr:`3965`: Added the ``scale_stroke`` boolean parameter to :meth:`.VMobject.scale`
+
+
+* :pr:`3974`: Made videos embedded in Google Colab by default
```

Comment:
I forgot to talk about this in my previous review. Might this be also considered a highlight, since now we don't have to type `config["media_embed"] = True` in Colab?